### PR TITLE
[CCP-177] Introduced pagination in concept types screen

### PR DESCRIPTION
**Pull request by mkarthik90**
Server side pagination is introduced in concept types screen. Pagination will be introduced for concept search screen in next PR.

### CODE REVIEW

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptTypesManager.java [hunk 1]
```diff
@@ -15,6 +17,9 @@
 	@Autowired
 	private TypeDatabaseClient client;
 
+    @Value("${default_page_size}")
+    private Integer defaultPageSize;
+
```
Comment (mkarthik90) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptTypesManager.java]: I think we can format all the files once. The current setting has "spaces" formatting but existing file is formatted using tab spaces.

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptTypesController.java [hunk 2]
```diff
@@ -27,12 +31,22 @@
 	 * @param model
 	 *            A generic model holder for Servlet
 	 * @return String value to redirect user to concept type list page
+	 * @throws SecurityException 
+	 * @throws NoSuchFieldException 
 	 */
 	@RequestMapping(value = "auth/concepttype")
-	public String prepateShowConceptTypes(ModelMap model) {
+    public String prepateShowConceptTypes(ModelMap model, @RequestParam(defaultValue = "1") String page,
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptTypesController.java]: prepate?

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptTypesController.java [hunk 3]
```diff
@@ -27,12 +31,22 @@
...
+            @RequestParam(defaultValue = IConceptDBManager.DESCENDING + "") String sortDir,
+            @RequestParam(defaultValue = "typeId") String sortColumn) throws NoSuchFieldException, SecurityException {
 
 		ConceptType[] types = conceptTypesManager.getAllTypes();
 
+		int pageInt = new Integer(page);
+        int sortDirInt = new Integer(sortDir);
+        int pageCount = conceptTypesManager.getPageCount();
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptTypesController.java]: call when needed

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
types are not sorted by there names anymore

**Comment by mkarthik90** [comment]
@jdamerow  Addressed review comments

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
please format the files that are misformatted and then we should be good to go

**Comment by mkarthik90** [comment]
@jdamerow Formatted the files.

**Pull request merged**